src/style.css

```
@@ -1,3 +1,76 @@
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+.logo-image {
+  width: 40px;
+  height: 40px;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  display: block;
+}
+
+.nav-link {
```

```suggestion
.nav__link {
```

Name of element in BEM should be with a double underscore, fix it everywhere